<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventMonitorTab</class>
 <widget class="QWidget" name="EventMonitorTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>447</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="QGroupBox" name="_emailGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>E-Mail Notification</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="_emailNotify">
        <property name="text">
         <string>N&amp;otify:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="_emailRecipients">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="_equationGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Expression</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>Vectors:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Operators:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>_equationOperator</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="_equationOperator">
        <item>
         <property name="text">
          <string>+</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>*</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>^</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>e</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>=</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&lt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&lt;=</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&gt;=</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&amp;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>|</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>!</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>STEP()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ABS()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SQRT()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CBRT()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SIN()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COS()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TAN()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ASIN()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ACOS()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ATAN()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SEC()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CSC()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COT()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SINH()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COSH()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TANH()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EXP()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LN()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LOG()</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Kst::ScalarSelector" name="_scalarSelector" native="true"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel3">
        <property name="text">
         <string>Scalars:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="_equation"/>
      </item>
      <item row="1" column="1">
       <widget class="Kst::VectorSelector" name="_vectorSelector" native="true"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="textLabel3_2">
        <property name="text">
         <string>Expression:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>_equation</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="textLabel3_2_3">
        <property name="text">
         <string>Description:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>_description</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="_description"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="_KstScriptGroupBox">
     <property name="title">
      <string>KstScript</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="_executeScript">
        <property name="text">
         <string>Execute:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="_script">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="_ELOGGroupBox">
     <property name="title">
      <string>ELOG</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="_ELOGNotify">
        <property name="text">
         <string>No&amp;tify:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_ELOGConfigure">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Configure...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>241</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="_logButtonGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Kst Debug Log</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="5">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>120</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="_debugLogWarning">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Warning</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="_debugLogError">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Error</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="_debugLogNotice">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Notice</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="_debugLog">
        <property name="text">
         <string>&amp;Log as:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Kst::VectorSelector</class>
   <extends>QWidget</extends>
   <header>vectorselector.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::ScalarSelector</class>
   <extends>QWidget</extends>
   <header>scalarselector.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_equationOperator</tabstop>
  <tabstop>_vectorSelector</tabstop>
  <tabstop>_scalarSelector</tabstop>
  <tabstop>_equation</tabstop>
  <tabstop>_description</tabstop>
  <tabstop>_debugLog</tabstop>
  <tabstop>_debugLogNotice</tabstop>
  <tabstop>_debugLogWarning</tabstop>
  <tabstop>_debugLogError</tabstop>
  <tabstop>_emailNotify</tabstop>
  <tabstop>_emailRecipients</tabstop>
  <tabstop>_ELOGNotify</tabstop>
  <tabstop>_ELOGConfigure</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_emailNotify</sender>
   <signal>toggled(bool)</signal>
   <receiver>_emailRecipients</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_ELOGNotify</sender>
   <signal>toggled(bool)</signal>
   <receiver>_ELOGConfigure</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_executeScript</sender>
   <signal>toggled(bool)</signal>
   <receiver>_script</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_debugLog</sender>
   <signal>toggled(bool)</signal>
   <receiver>_debugLogNotice</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_debugLog</sender>
   <signal>toggled(bool)</signal>
   <receiver>_debugLogWarning</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_debugLog</sender>
   <signal>toggled(bool)</signal>
   <receiver>_debugLogError</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
